### PR TITLE
et_feeder: Bugfix in attr parsing in et_feeder

### PR DESCRIPTION
**Pull request by TaekyungHeo**
## Summary
This pull request updates the `et_feeder` module to fix a critical parsing issue. The problem stemmed from two main sources: inadequate support in protobuf for the `oneof` boolean value, and a bug within the `ETFeederNode::assign_attr_val` function.

## Test Plan
```
$ cd ~/param
$ cd param/train/comms/pt
$ pip install .
$ cd ../../compute/python
$ pip install -r requirements.txt
$ python setup.py install
$ python tools/trace_link.py --pytorch-et-file ~/llama_pytorch_et/llama_et_0.json --kineto-file ~/llama_kineto/worker0_step_12.1697596714999.pt.trace.json --output-file ~/rank0.json

$ cd ~/charka
$ pip install .
$ python3 -m chakra.et_converter.et_converter --input_type PyTorch --input_filename ~/rank0.json --output_filename ~/rank0.chakra --num_dims 1 
$ cp ~/rank0.chakra chakra.0.et && cp chakra.0.et chakra.1.et

$ cd ~/astra-sim
$ ./build/astra_analytical/build.sh
$ ./build/astra_analytical/build/bin/AstraSim_Analytical_Congestion_Unaware\
  --workload-configuration=/Users/theo/chakra/chakra\
  --system-configuration=./inputs/system/Switch.json \
  --network-configuration=./inputs/network/analytical/Switch.yml \
  --remote-memory-configuration=./inputs/remote_memory/analytical/no_memory_expansion.json
ring of node 0, id: 0 dimension: local total nodes in ring: 2 index in ring: 0 offset: 1total nodes in ring: 2
ring of node 0, id: 0 dimension: local total nodes in ring: 2 index in ring: 0 offset: 1total nodes in ring: 2
ring of node 0, id: 0 dimension: local total nodes in ring: 2 index in ring: 0 offset: 1total nodes in ring: 2
ring of node 0, id: 0 dimension: local total nodes in ring: 2 index in ring: 0 offset: 1total nodes in ring: 2
sys[0] finished, 7269182000 cycles
sys[1] finished, 7269182000 cycles
```

### CODE REVIEW

File: et_feeder/et_feeder_node.h [hunk 1]
```diff
@@ -49,7 +49,7 @@ class ETFeederNode {
 
   uint64_t id_;
   std::string name_;
-  bool is_cpu_op_;
+  uint32_t is_cpu_op_;
```
Comment (JoongunPark) [inline et_feeder/et_feeder_node.h:52]: Why is_cpu_op should be uint32_t?

Comment (JoongunPark) [reply et_feeder/et_feeder_node.h:52]: Was it because "inadequate support in protobuf for the oneof boolean value"?

**Comment by JoongunPark** [comment]
I think commit "https://github.com/mlcommons/chakra/pull/18/commits/650f27616fd03cb55022a7e4c65d9432a62a1878" is duplicated with the commit in the another PR. Please check!
Other than that, I think PR should be accepted.

File: et_feeder/et_feeder_node.h (hunk 1 above)
Comment (TaekyungHeo) [reply et_feeder/et_feeder_node.h:52]: It seems that protobuf does not support oneof boolean values properly on some systems. That's why we are replacing it with uint32_t.

**Comment by TaekyungHeo** [comment]
> I think commit "[650f276](https://github.com/mlcommons/chakra/commit/650f27616fd03cb55022a7e4c65d9432a62a1878)" is duplicated with the commit in the another PR. Please check! Other than that, I think PR should be accepted.

Thanks, this commit appears because this PR is based on another PR. PRs should be accepted in order.

File: et_feeder/et_feeder_node.h (hunk 1 above)
Comment (JoongunPark) [reply et_feeder/et_feeder_node.h:52]: Got it! Thank you.

**Review by srinivas212 — APPROVED** [review]

**Pull request merged**